tests/console/wpa_supplicant.pm

```diff
@@ -37,6 +37,8 @@ sub run {
     zypper_call 'in wpa_supplicant hostapd iw dnsmasq unzip';
     assert_script_run 'cd $(mktemp -d)';
     assert_script_run('curl -L -s ' . data_url('wpa_supplicant') . ' | cpio --make-directories --extract && cd data');
+    # AppArmor doesn't allow config files from arbitrary locations
+    script_run('aa-disable hostapd');
```

Comment:
We could also adopt apparmor profile like:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12445/files#diff-127523a550f20e15c0c17586e480d9c2d4218726746e55f53ce804e3436d5560R77

Because we do not have flag like `allways_rollback` and disabling aa would affect later coming test-modules.

Reply:
Let's keep it simple for now, DeepthiYV is still learning and provided a nice and easy fix for this test failure on Tumbleweed. 
I believe all of those issues are really outside the scope of a simple fix and should be treated separately.

Reply:
I don't find this right either. You say you added permission But I see `aa-disable` command. @cfconrad WDYT?

Reply:
isn't it outdated....

Reply:
yes it is, the final solution was to copy the `adopt_aparmor()` function.